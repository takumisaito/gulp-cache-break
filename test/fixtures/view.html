<meta charset="utf-8">
<title>CACHE BREAK</title>
<link rel="stylesheet" href="css/main.css">
<!--[if lt IE 9]>
  <script src="js/html5shiv-printshiv.js"></script>
<![endif]-->
<script defer src="js/libs.js"></script>
<script defer src="js/main.js"></script>