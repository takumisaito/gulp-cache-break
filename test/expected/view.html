<meta charset="utf-8">
<title>CACHE BREAK</title>
<link rel="stylesheet" href="css/main.css?rel=test_query_a">
<!--[if lt IE 9]>
  <script src="js/html5shiv-printshiv.js"></script>
<![endif]-->
<script defer src="js/libs.js"></script>
<script defer src="js/main.js?rel=test_query_a"></script>